usbrelay_py/setup.py

```diff
@@ -1,9 +1,11 @@
 from setuptools import setuptools, setup, Extension
+exec(open('../LIBVER.in').read())
+
 
 module1 = Extension(
     'usbrelay_py',
-    libraries= ['usbrelay'],
-    library_dirs= ['./','/usr/lib','/usr/lib64','/usr/lib/x86_64-linux-gnu','/usr/lib/aarch64-linux-gnu','/usr/lib/arm-linux-gnueabihf','..'],
+    libraries= [':libusbrelay.so.' + str(USBMAJOR)],
+    library_dirs= ['./','/usr/lib','/usr/lib64','/usr/lib/x86_64-linux-gnu','/usr/lib/aarch64-linux-gnu','/usr/lib/arm-linux-gnueabihf','../'],
```

Comment:
I think most of those libraries are not necessary. This is the reason why your build tried to link against wrong library.
```suggestion
    library_dirs= ['../'],
```